imgee/__init__.py

```diff
@@ -48,4 +44,6 @@ def error403(error):
     app.config['MEDIA_DOMAIN'] = app.config['MEDIA_DOMAIN'].split(':', 1)[1]
 mkdir_p(app.config['UPLOADED_FILES_DEST'])
```

Comment:
Just noticed this `mkdir` call. The should be in `manage.py` as say `manage.py init`. It should not be a runtime check.